File: src/main/java/entites/Vehicule.java
package entites;

//import java.sql.Date;
import javafx.scene.control.ChoiceBox;
import javafx.scene.control.TextField;
import utils.DataSource;
//import MyConnection; // Import your MyConnection class

public class Vehicule {
    private int num_v;
    private String type;
    private int capacite;
    private int prixuni;
    private int num_ch;
    private DataSource MyConnection;

    public Vehicule(int num_v, String type, int capacite, int prixuni, int num_ch) {
        this.num_v = num_v;
        this.type = type;
        this.capacite = capacite;
        this.prixuni = prixuni;
        this.num_ch = num_ch;

    }

    public Vehicule(TextField numV, ChoiceBox<String> type, TextField capacite, TextField prixuni, TextField numCh) {
    }

    public int getNum_v() {
        return num_v;
    }

    public void setNum_v(int num_v) {
        this.num_v = num_v;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getCapacite() {
        return capacite;
    }

    public void setCapacite(int capacite) {
        this.capacite = capacite;
    }

    public int getPrixuni() {
        return prixuni;
    }

    public void setPrixuni(int prixuni) {
        this.prixuni = prixuni;
    }

    public int getNum_ch() {
        return num_ch;
    }

    public void setNum_ch(int num_ch) {
        this.num_ch = num_ch;
    }

    @Override
    public String toString() {
        return "Vehicule{" +
                "num_v=" + num_v +
                ", type='" + type + '\'' +
                ", capacite=" + capacite +
                ", prixuni=" + prixuni +
                ", num_ch=" + num_ch +
                '}';
    }

    }

File: Vehicule.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Vehicule
 *
 * @ORM\Table(name="vehicule", indexes={@ORM\Index(name="num_ch", columns={"num_ch"})})
 * @ORM\Entity
 */
class Vehicule
{
    /**
     * @var int
     *
     * @ORM\Column(name="num_v", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $numV;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=100, nullable=false)
     */
    private $type;

    /**
     * @var int
     *
     * @ORM\Column(name="capacite", type="integer", nullable=false)
     */
    private $capacite;

    /**
     * @var int
     *
     * @ORM\Column(name="prixuni", type="integer", nullable=false)
     */
    private $prixuni;

    /**
     * @var int
     *
     * @ORM\Column(name="num_ch", type="integer", nullable=false)
     */
    private $numCh;

    public function getNumV(): ?int
    {
        return $this->numV;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(string $type): static
    {
        $this->type = $type;

        return $this;
    }

    public function getCapacite(): ?int
    {
        return $this->capacite;
    }

    public function setCapacite(int $capacite): static
    {
        $this->capacite = $capacite;

        return $this;
    }

    public function getPrixuni(): ?int
    {
        return $this->prixuni;
    }

    public function setPrixuni(int $prixuni): static
    {
        $this->prixuni = $prixuni;

        return $this;
    }

    public function getNumCh(): ?int
    {
        return $this->numCh;
    }

    public function setNumCh(int $numCh): static
    {
        $this->numCh = $numCh;

        return $this;
    }


}
